doc/source/rllib/rllib-rlmodule.rst

```diff
@@ -408,7 +440,9 @@ There are two possible ways to extend existing RL Modules:
 Checkpointing RL Modules
 ------------------------
 
-RL Modules can be checkpointed with their two methods :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.save_to_checkpoint` and :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.from_checkpoint`.
+RL Modules can be checkpointed via their :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.save` method
+and can be constructed from a checkpoint via the :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.from_checkpoint` method.
+
```

Comment:
Maybe adding that `RLModule`s can be used as single instances apart of the rest of RLlib?